andela/ah-backend-stark
# #160584019 get user reading stats

#### What does this PR do?
Allows users to see how many articles they have read and written
#### Description of Task to be completed?
Update database with reading stats and writing stats on a user's profile
#### How should this be manually tested?
After pulling the branch
- run the following commands to update the models
`python manage.py makemigrations`
`python manage.py migrate`
- run the application on Postman using the following command
`python manage.py runserver`
- use the following endpoints to register and log in a user and then post an article
`POST 'http://127.0.0.1:8000/users/'`
`POST 'http://127.0.0.1:8000/users/login/'`
`POST 'http://127.0.0.1:8000/articles/'`
- confirm that the user's writing stats have been updated using the following endpoint
`GET 'http://127.0.0.1:8000/profile/<str:username>/'`
- log another user in and use the following endpoint to get the previously created article
`GET 'http://127.0.0.1:8000/articles/<str:slug>'`
- confirm that this user's reading stats have been updated using the following endpoint
`GET 'http://127.0.0.1:8000/profile/<str:username>/'`
#### What are the relevant pivotal tracker stories?
#160584019 - Users should be able to see their reading stats
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40589388/47707703-88245b80-dc3d-11e8-8e17-af5c7f6ec892.png)
